extensions/wikia/CoppaTool/CoppaToolController.class.php

```diff
@@ -140,7 +140,7 @@ public function renameIPAddress() {
 
 		$ipAddr = IP::sanitizeIP( $ipAddr );
 
-		$wikiIDs = RenameUserHelper::lookupIPActivity( $ipAddr );
+		$wikiIDs = RenameIPHelper::lookupIPActivity( $ipAddr );
 
 		$taskParams = [
 			'requestor_id' => $this->wg->User->getID(),
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)

[project-id]: wikia:app